src/buildfile.js

```diff
@@ -17,6 +17,7 @@ exports.base = [{
 
 exports.workerExtensionHost = [entrypoint('vs/workbench/services/extensions/worker/extensionHostWorker')];
 exports.workerNotebook = [entrypoint('vs/workbench/contrib/notebook/common/services/notebookSimpleWorker')];
+exports.workerLanguageDetection = [entrypoint('vs/workbench/services/languageDetection/common/languageDetectionSimpleWorker')];
```

Comment:
Is there a way I could include the `@vscode/vscode-languagedetection` npm package here?